src/Dealership.java

```diff
@@ -0,0 +1,45 @@
+package src;
+import java.util.ArrayList;
+
+public class Dealership {
+    private int dealerID;
+    private boolean vehicleAcquisition;
+    private ArrayList<Vehicle> vehicleInventory;
+
+    public Dealership(int newDealer) {
+        dealerID = newDealer;
+        vehicleAcquisition = true;
+        vehicleInventory = new ArrayList<Vehicle>();
```

Comment:
I think in class we talked about how a map might be better since it would be easier to access/manipulate the vehicles based on vehicleID rather than having to implement a search method.

With how it is now there may be multiple vehicles with the same vehicleID which defeats the purpose of the ID. 

You could have something like:

private Map<Integer,Vehicle> vehicleInventory;

and then in the constructor initialize it 
this.vehicleInventory= new HashMap<>();

javadoc for Map: https://docs.oracle.com/javase/8/docs/api/java/util/Map.html?is-external=true

Reply:
Thanks for the reminder, I'll make adjustments to use maps instead with an integer key to start with.